ui/js/controllers/main.js

```diff
@@ -406,6 +406,18 @@ treeherderApp.controller('MainCtrl', [
             );
         };
 
+        $scope.fromChangeValue = function() {
+            var url = window.location.href;
+            url = url.replace("&fromchange=" + $location.search()["fromchange"], "")
```

Comment:
Missing semi-colon (and below), which is what is causing the linter job to fail :-)